templates/security/cve/filters/releases-filter.html

```diff
@@ -0,0 +1,32 @@
+<form>
+  <label class="p-checkbox">
+    <input type="checkbox" aria-labelledby="allMaintainedReleases" class="p-checkbox__input" />
+    <span class="p-checkbox__label" id="allMaintainedReleases">All maintained releases</span>
+  </label>
+  <label class="p-checkbox">
+    <input type="checkbox" aria-labelledby="22.10" class="p-checkbox__input" />
+    <span class="p-checkbox__label" id="22.10">22.10 kinetic</span>
+  </label>
+  <label class="p-checkbox">
+    <input type="checkbox" aria-labelledby="22.04" class="p-checkbox__input" />
+    <span class="p-checkbox__label" id="22.04">22.04 LTS jammy</span>
+  </label>
+  <label class="p-checkbox">
+    <input type="checkbox" aria-labelledby="20.04" class="p-checkbox__input" />
+    <span class="p-checkbox__label" id="20.04">20.04 LTS focal</span>
+  </label>
+  <p class="p-text--small-caps" style="padding-left: 2rem;">Still maintained with<br/> Ubuntu Pro</p>
+  <label class="p-checkbox">
+    <input type="checkbox" aria-labelledby="18.04" class="p-checkbox__input" />
+    <span class="p-checkbox__label" id="18.04">18.04 LTS bionic</span>
+  </label>
+  <label class="p-checkbox">
+    <input type="checkbox" aria-labelledby="16.04" class="p-checkbox__input" />
+    <span class="p-checkbox__label" id="16.04">16.04 LTS xenial</span>
+  </label>
+  <label class="p-checkbox">
+    <input type="checkbox" aria-labelledby="14.04" class="p-checkbox__input" />
+    <span class="p-checkbox__label" id="14.04">14.04 LTS trusty</span>
+  </label>
+</form>
+<a>Show unmaintained releases</a>
```

Comment:
Where is this supposed to link to? i assume a new page that isn't built yet

Reply:
@juanruitina could you confirm where this, the "What is Ubuntu Priority" and "What do statuses mean?" links are supposed to link to?